sgx-types/src/secs.rs

```diff
@@ -65,6 +65,12 @@ impl Spec {
         let max_size: u64 = 1 << (res.edx >> 8 as u8) as u64;
         Some(NonZeroU64::new(max_size).unwrap())
     }
+
+    /// Returns the size of the SSA in 4k pages, rather than bytes. For more, see documentation
+    /// on ssa::StateSaveArea::frame_size().
+    pub fn ssa_size() -> Option<NonZeroU32> {
+        StateSaveArea::frame_size()
+    }
```

Comment:
It is possible that the consumer of this library doesn't want to use our `StateSaveArea` type. That makes the size of the SSA a property of the `StateSaveArea` type rather than the `Spec` type. Please move the method there.

Reply:
However, I would take an `impl Default for Spec` where the defaults use the `StateSaveArea` and the maximum enclave size.

Reply:
Not sure why this isn't marked as `Outdated`, but I did make this change.